vtrace/envitools.py

```diff
@@ -130,13 +127,25 @@ def platformGetThreads(self):
     def platformGetFds(self):
         return []  # FIXME perhaps tie this into magic?
 
+    def archGetRegCtx(self):
```

Comment:
Well that's awkward. I'm guessing there weren't any consumers of this so we didn't notice this till now?

Reply:
correct